_topics/compact-block-filters.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Compact block filters
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - BIP157
+  - BIP158
+  - Neutrino protocol
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Privacy Enhancements
+  - Lightweight Client Support
+  - P2P Network Protocol
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Compact block filters** are a condensed representation of the
+  contents of a block that allow wallets to determine whether the block
+  contains any transactions affecting the wallet.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  A full node with a copy of a block and the data necessary to validate
+  that block uses [BIP158][] to create a Golomb-Rice Coded Set (GCS) of
+  data from that block (such as the prevout script from each input spent
+  in the block and the scriptPubKey for each output created in the
+  block).  The GCS (called a filter) is then distributed to wallets
+  (such as via the P2P protocol as describide in [BIP157][]), allowing
+  them to search for any matches to each wallet's scripts.  If a match
+  is found, the wallet can then download the corresponding block to
+  access any relevant transactions.
+
+    The GCS mechanism guarantees that a properly-coded wallet will find
+    any transactions matching its scripts, but it may also find some
+    false-positive matches for which it will need to download and scan
+    the block despite the block not containing any transactions relevant
+    to the wallet.
+
+    The BIP157/158 protocol is sometimes called "Neutrino" after the
```

Comment:
Just the "Neutrino" alias or the whole paragraph?  (The whole paragraph would be hard because the current first paragraph of the extendend summary describes BIP157 and BIP158 individually.)

Reply:
I like that whole 'is sometimes called "Neutrino"' paragraph as the first one since it seems most summary level to me. Im not strong on this suggestion, your call.